src/components/library/LibraryMangaGrid.tsx

```diff
@@ -63,15 +64,12 @@ const filterManga = (
             }
             return acc;
         }, {});
-        return Object.values(unique);
+        filteredManga = Object.values(unique);
```

Comment:
this commit reverts #226.
not sure how everyone else sees this, but I'd like to be able to search my whole library and not just the filterted one (e.g. downloaded, unread), since otherwise, if you want to open a manga that is not included in the filters, you have to unset the filters everytime you search for something like that (and set them again after you've found the manga)

Reply:
Is this not how the app works? If you want to search something that is not downloaded then you remove the filter and search for it, right?

It felt like a bug that my filters were not being honored when I searched for my manga

I believe we should stick close to the app implementation. I think we should not deviate too far from the tachiyomi app. If setting and unsetting the filters are cumbersome or buggy we could improve the experience there.  I will adhere to the consensus. But, I think we should keep this.

Reply:
I never used the app (don't have an android), so I can't say.

I guess it's preference. I expected it to search my whole library (or at least the category, since whole library is not possible).
since I know what I'm searching for (a specific title), I wouldn't want a filter to mess with it.

if it should work like the app, what about adding some kind of flag to search all mangas/ignore filters?

Reply:
That sounds like a good workaround, but for that we need to clarify the following details
1. Whether this preference will be stored in localStorage or sever
2. Where can users find this preference(Settings page or somewhere else)
3. Will it be on/off by default
4. Any other solutions like displaying a fab while searching and if filters are active, to quicky disable the filters(either for that search or permanently)
5. Should it be done in this PR or another(I prefer another PR)

Reply:
@schroda any opinions on these? I'd like to get this PR reviewed

Reply:
1. I'd say server as well
2. maybe the library "settings" (filter, sort, ui), or a settings button inside the search, which is only visible when search is open?
3. should be the same as the app then, so disabled
4. I wouldn't mess with the filters, since it's also not really required for it to work
5. yes, should be done in another pr

Reply:
@schroda I have created a new PR https://github.com/Suwayomi/Tachidesk-WebUI/pull/242 to find a solution for the issue in this thread